app/src/main/java/org/novinger/jason/demoreader/datamodels/Article.java

```diff
@@ -0,0 +1,67 @@
+package org.novinger.jason.demoreader.datamodels;
+
+/**
+ * Created by jason on 4/27/16.
+ */
+public class Article {
+    private String id;
+    private int realId;
+    private String headline;
+    private String slug;
+    private Author[] authors;
+    private String summary;
+    private String url;
+
+    public Article() {}
+
+    public Article(String id, String headline, String slug, Author[] authors, String url, String summary) {
+        this.id = id;
```

Comment:
since this is a Gson class i might actually just do what you did...  i tend to let Gso POJOs be ugly :/